api/alert_channels.go

```diff
@@ -83,14 +83,16 @@ const (
 	EmailUserAlertChannelType
 	SlackChannelAlertChannelType
 	AwsS3AlertChannelType
+	AWSCloudwatchAlertChannelType
```

Comment:
We try, as much as possible to use the exact name that the API returns
```suggestion
	CloudwatchEbAlertChannelType
```